app/components/add-ssh-key.js

```diff
@@ -56,16 +58,16 @@ export default Component.extend({
     this.set('valueError', false);
 
     if (this.isValid()) {
-      const sshKey = this.get('model');
+      const sshKey = this.model;
       sshKey.setProperties({
-        description: this.get('description'),
-        value: this.get('value')
+        description: this.description,
+        value: this.value
       });
 
       try {
         yield sshKey.save();
         this.reset();
-        return this.sendAction('sshKeyAdded', sshKey);
+        return this.sshKeyAdded(sshKey);
```

Comment:
Out of curiosity, is this doing anything or is just a noop?

Reply:
The `sshKeyAdded` is a method that can be assigned from the outside as a property, when the component is used, e.g.:
```hbs
{{add-ssh-key sshKeyAdded=(action 'onSshKeyAdded')}}
```

Reply:
The no-op is just a stub so that the action won't fail if the property is not assigned. This makes components more flexible. And also explicitly defines public interface of a component.